Update project docs (#571)
elastic/integrations: README.md

[![Build Status](https://beats-ci.elastic.co/job/ingest-manager/job/integrations/job/master/badge/icon)](https://beats-ci.elastic.co/job/ingest-manager/job/integrations/job/master/)

# Elastic Integrations

> This repository contains sources of the Elastic Integrations - packages defining data streams to observe the same product (logs, metrics, traces).

Reviewer: ```suggestion
This repository contains sources for Elastic Integrations. Each Elastic Integration is an Elastic Package that defines how to observe a specific product with the Elastic Stack.
```

> ```suggestion

Reviewer 2: I believe this is a leftover from the GH suggestion tool, let's remove it

Author: I knew I missed this label somewhere. Thanks, fixed!

> A package may contain configuration of the [Elastic-Agent](https://github.com/elastic/beats/tree/master/x-pack/elastic-agent), dashboards, visualizations,
> images, tests, documentation, etc.

Reviewer: ```suggestion
```suggestion
An Elastic Package may define configuration for the [Elastic Agent](https://github.com/elastic/beats/tree/master/x-pack/elastic-agent) as well as assets (such as Kibana dashboards and Elasticsearch index templates) for the Elastic Stack. It should also define documentation about the package. Finally, a package may also define tests to ensure that it is functioning as expected.
```
```

> The package format is described in the [Package Spec](https://github.com/elastic/package-spec). The repository is also used

Reviewer: ```suggestion
Elastic Packages have a certain, well-defined structure. This structure is described by the [Package Specification](https://github.com/elastic/package-spec). The repository is also used
```

> for discussions/extending the specification (with proposals).

Reviewer: ```suggestion
for discussions about extending the specification (with proposals).
```

> The Elastic Integrations are distributed with the [Package Storage](https://github.com/elastic/package-storage)
> and exposed for [Kibana](https://github.com/elastic/kibana) via the [Package Registry](https://github.com/elastic/package-registry/).

Reviewer: ```suggestion
While this repository contains _sources_ for Elastic Integrations, _built_ Elastic Integrations are stored in the [Package Storage](https://github.com/elastic/package-storage) repository and served up via the [Package Registry](https://github.com/elastic/package-registry/). The Fleet UI in Kibana connects to the Package Registry and allows users to discover, install, and configure Elastic Packages.
```

> The official builder tool for the Elastic Integrations is the [elastic-package](https://github.com/elastic/elastic-package).

Reviewer: ```suggestion
The official builder tool for Elastic Integrations is [`elastic-package`](https://github.com/elastic/elastic-package).
```

## Contributing

> Please review the [CONTRIBUTING](CONTRIBUTING.md) guide to learn how to build and develop packages, understand the release procedure and

Reviewer: ```suggestion
Please review the [Contributing Guide](CONTRIBUTING.md) to learn how to build and develop packages, understand the release procedure and
```
